.storybook/themes/commerce.scss

```diff
@@ -627,6 +627,23 @@
 						),
 					),
 				),
+				product-information: (
+					components: (
+						heading: (
+							color: var(--global-black),
+							font-size: var(--global-font-size-9),
```

Comment:
nice -- closest one based on designs of 26

Reply:
looks like line height should be 32px -- and it's 20px now on desktop


![Screen Shot 2022-07-21 at 06 28 10](https://user-images.githubusercontent.com/5950956/180192710-49ec04fe-f348-4a16-8ab7-80c13f7f8757.png)